client/src/context/quiz/QuizState.js

```diff
@@ -49,7 +49,7 @@ export const getQuizzes = async(dispatch) => {
 //get quiz
 export const getQuiz = async (dispatch, quiz) => {
     try {
-		const res = await axios.get(route + `/${quiz._id}`);
+		const res = await axios.get(route + `/${quiz}`);
```

Comment:
I made changes on the quiz dashboard to get the quiz by quiz_id from param, I didn't see any other changes in your code that utilizes this, not sure if you are using this anywhere else in the code.